src/js/components/Form/__tests__/Form-test-uncontrolled.js

```diff
@@ -127,25 +127,31 @@ describe('Form uncontrolled', () => {
   });
 
   test('errors', () => {
-    const component = renderer.create(
-      <Grommet>
-        <Form errors={{ test: 'missing' }}>
-          <FormField name="test" />
-        </Form>
-      </Grommet>,
-    );
+    let component;
+    renderer.act(() => {
```

Comment:
Can you add a brief comment on why we need `act()`? I suspect we might forget.

Reply:
Done

Reply:
My comment on `act` did not come through the first time, so I'll try again - sorry about that.

You might be able to clean up the usage of `act` and `component = renderer.create(...)` with a more updated API of jest and '@testing-library/react', which is simply using the concept of `render` as follows

```
  const { container } = render(
      <Grommet>
         ...
      </Grommet>,
    );
```
    
`act` and `renderer` are limited in their functionality, and will probably be deprecated at one point, it is better not to introduce any new ones.

Reply:
I did actually try that first and it didn't help but I can double check and see if that's doable.

Reply:
Done